File: Kattis/keysphonewallet/keysphonewallet.cpp
#include <iostream>
#include <vector>
#include <string>

int main() {
    int N;
    std::cin >> N;
    std::vector<std::string> inventory = {"keys", "phone", "wallet"};

    while (N > 0) {
        std::string item;
        std::cin >> item;

        for (int i = 0; i < inventory.size(); i++)  {
            if (item == inventory[i])
                inventory.erase(inventory.begin() + i);
        }
        N--;
    }

    if (inventory.empty()) {
        std::cout << "ready\n";
    }
    else {
        for (std::string essentials : inventory) {
            std::cout << essentials << "\n";
        }
    }
    return 0;
}

File: Kattis/keysphonewallet/keysphonewallet.py
N = int(input())
inventory = ["keys", "phone", "wallet"]

while N > 0:
    item = input()

    for essentials in inventory:
        if item == essentials:
            inventory.remove(essentials)
    N -= 1

if not inventory:
    print("ready")
else:
    for essentials in inventory:
        print(essentials)